src/components/view/sources/sources.ts

```diff
@@ -0,0 +1,50 @@
+import { Source } from '../../models/index.model';
+import './sources.css';
+
+class Sources {
+  private filter: string = 'A';
+  draw(data: Array<Source>) {
+    const letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
+    const fragment = document.createDocumentFragment();
+    const sourceItemTemp: HTMLTemplateElement | null = document.querySelector('#sourceItemTemp');
+
+    const drawData = () => {
+      data
+        .filter((el) => el.name.startsWith(this.filter))
+        .forEach((item) => {
+          if (!sourceItemTemp) {
+            return;
+          }
+          const sourceClone = sourceItemTemp.content.cloneNode(true);
+          if (sourceClone instanceof DocumentFragment) {
```

Comment:
do you need to verify intance type, are there options to receive anything except of `DocumentFragment `?